hasktorch/src/Torch/Typed/Native.hs

```diff
@@ -2350,7 +2351,7 @@ lstm_cell :: Tensor dtype '[batchSize, inputDim]
   -> (Tensor dtype '[batchSize, hiddenSize],Tensor dtype '[batchSize, hiddenSize])
 lstm_cell _input (_cc, _hc) _w_ih _w_hh _b_ih _b_hh = unsafePerformIO $ (cast6 ATen.lstm_cell_tltttt) _input _hx _w_ih _w_hh _b_ih _b_hh
   where 
-    _hx = [toDynamic _cc, toDynamic _hc]
+    _hx = [_cc, _hc] :: [Tensor dtype '[batchSize, hiddenSize]]
```

Comment:
yes, toDynamic shouldn't be necessary here. I'm surprised that you have to annotate the type. does it type-check but not run?

Reply:
The test fails with an ambiguous type. I think due to the `IsList` instance of tensor.  We'll see if the annotation clears it up.